azure/microservice-ecommerce/blueprint.yaml

```diff
@@ -0,0 +1,201 @@
+  - name: AKSRegion
+    type: Select
+    description: "Select the AKS region for the cluster:"
+    dependsOnTrue: ProvisionCluster
+    options:
+    - eastasia
+    - southeastasia
+    - centralus
+    - eastus
+    - eastus2
+    - westus
+    - northcentralus
+    - southcentralus
+    - northeurope
+    - westeurope
+    - japanwest
+    - japaneast
+    - brazilsouth
+    - australiaeast
+    - australiasoutheast
+    - southindia
+    - centralindia
+    - westindia
+    - canadacentral
+    - canadaeast
+    - uksouth
+    - ukwest
+    - westcentralus
+    - westus2
+    - koreacentral
+    - koreasouth
+    - francecentral
+    - francesouth
+    - australiacentral
+    - australiacentral2
+    - southafricanorth
+    - southafricawest
+
+    # ############################################################################
+  # Authentication
+  # ############################################################################
+  - name: SubscriptionID
+    type: Input
+    description: "Enter you Azure subscription id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: TenantID
+    type: Input
+    description: "Enter your Azure tenant id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: ClientID
+    type: Input
+    description: "Enter your Azure client id:"
+    dependsOnTrue: ProvisionCluster
+
+  - name: ClientSecret
+    type: Input
+    description: "Enter your Azure client secret:"
+    secret: true
+    dependsOnTrue: ProvisionCluster
+
+  # ############################################################################
+  # For the Terraform backend storage
+  # ############################################################################
+  - name: ResourceGroup
+    # Resource group names only allow alphanumeric characters, periods, underscores, hyphens and parenthesis and cannot end in a period
+    type: Input
+    description: "Enter a new name for the Resource Group that will be created to house this cluster:"
+    pattern: "^[\\w._()-]{2,50}[\\w_()-]$"
+    dependsOnTrue: ProvisionCluster
+
+  - name: StorageAccountName
+    type: Input
+    description: "Enter the Storage Account where Terraform will store its state (see README.md step 3.2):"
+    dependsOnTrue: ProvisionCluster
+
+  - name: StorageAccessKey
+    type: Input
+    description: "Enter the Storage Account key for accessing the storage (see README.md step 3.3):"
+    dependsOnTrue: ProvisionCluster
+
+  - name: StorageContainerName
+    type: Input
+    description: "Enter the Storage Container name (see README.md step 3.4):"
+    default: terraform-state
+    dependsOnTrue: ProvisionCluster
+  # ############################################################################
+
+  - name: ClusterName
+    type: Input
+    description: What name do you want to give the cluster?
+    pattern: "^[[:alpha:]][\\w-]{1,29}[\\w]$"
+    # dependsOnTrue: ProvisionCluster
+
+  - name: Namespace
+    type: Input
+    default: xl-demo
```

Comment:
I don't see a point in asking this as for the application we use, any other NS will not work, this makes more sense in basic blueprints that only creates clusters

Reply:
I have it as a prompt because of the [`gcp/microservice-ecommerce`](https://github.com/xebialabs/blueprints/blob/development/gcp/microservice-ecommerce/blueprint.yaml#L64) blueprint. I can make it a value or add a description like the `gcp/microservice-ecommerce` example. But if I make it a value, shouldn't I then update the `gcp/microservice-ecommerce` blueprint to also be a value?

Reply:
I think we should update both blueprint to make it a value

Reply:
Okay. Will do that now then.